tests/test_bayes_search.py

```diff
@@ -545,3 +545,318 @@ def loss_func(x: SweepRun) -> floating:
     np.random.set_state(seed)
     true = next_run(config, [r2])
     assert pred.config == true.config
+
+
+def test_gcp_issue_xl7xiuf6empym():
```

Comment:
xl7xiuf6empym 😂

but really, a description would help here.

Reply:
totally. xl7xiuf6empym is not so informative... will fix